### ISSUE TITLE
Remove omitempty for ContinueResponseBody.AllThreadsContinued

### ISSUE THREAD

**Issue by dbaumgarten** [google/go-dap#39]
The field ContinueResponseBody.AllThreadsContinued is marked with "omitempty" ( https://github.com/google/go-dap/blob/master/schematypes.go#L136 ).

This causes the json-encoder to omit the field if it's value is false (as false is the empty value for bools).

Unfortunately the spec says if the value is missing it is assumed to be true for backwards-compatibility. ( https://microsoft.github.io/debug-adapter-protocol/specification#Requests_Continue (see the comment of the field))

This makes it impossible to set this field to false from go.
If it is set to false, it is omitted and the client assumes it to be true.
( If it is true, the client accepts it as true. But this is not the case I am interested in).

Tl;dr: Remove omitempty from the field so it can actually be set to false.
